common/web/keyboard-processor/src/keyboards/defaultLayouts.ts

```diff
@@ -22,6 +22,7 @@ export type LayoutKey = {
   "nextlayer"?: string,
   "pad"?: string | number,
   "sk"?: LayoutKey[]
+  "default"?: boolean
```

Comment:
Is default just needed for ActiveKey? (wondering if LayoutKey and OSKKeySpec need it)

Reply:
https://github.com/keymanapp/keyman/blob/909331df62734410c11741c60d0b321632082430/common/web/keyboard-processor/src/keyboards/activeLayout.ts#L23

https://github.com/keymanapp/keyman/blob/909331df62734410c11741c60d0b321632082430/web/src/engine/osk/src/keyboard-layout/oskKey.ts#L13

When loaded, the touch-layout spec within a keyboard is generally an instance of type `LayoutKey` with all the properties and methods of `ActiveKey` copy-pasted onto it.

As for `OSKKeySpec`, after checking, it appears to only ever be constructed when a default is needed.  Otherwise, the `LayoutKey` is typecast to `OSKKeySpec`, and it works.